setup.cfg

```diff
@@ -4,8 +4,8 @@ python-tag = py3
 [tool:isort]
 profile = black
 
-[tool:flake8]
-max-line-length = 80
+[flake8]
+max-line-length = 88
```

Comment:
why 88?

Reply:
in pre-commit hook black is set to "-l 80", but black consider it as about 80 and try to be clever to avoid to create 3 lines instead 1.

I'll check to configure black in an harder way to avoid larger line than 80.

Reply:
I'm thinking to remove black as far I remember you dislike it.

I quite likes it because with it's very opinionated conventions it aims to produce less noise while reviewing code.